### PR TITLE
applications: nrf_desktop: HLK tests description

### PR DESCRIPTION
**Pull request by MarekPieta**
Change adds short HLK tests description to the documentation.

Jira:DESK-1104

### CODE REVIEW

**Review by pdunaj — APPROVED** [review]

File: applications/nrf_desktop/README.rst [hunk 1]
```diff
@@ -788,6 +788,12 @@ After building the application with or without :ref:`specifying the build type <
 
 After the connection is established and the device is bonded, you can use the mouse with the connected device.
 
+Windows HLK tests
+-----------------
+
+The nRF Desktop devices are confirmed to pass the tests from official playlist (:file:`HLK Version 1903 CompatPlaylist x86 x64 ARM64.xml`) required for compatibility with Windows 10 by Windows Hardware Compatibility Program.
+The tests were conducted using Windows Hardware Lab Kit.
+
 ----
```
Comment (greg-fer) [inline applications/nrf_desktop/README.rst:797]:
```suggestion

```
Please remove the divider. It is now automatically added to *** headings. (Check for other dividers in the file, please.)

File: applications/nrf_desktop/README.rst [hunk 2]
```diff
@@ -788,6 +788,12 @@ After building the application with or without :ref:`specifying the build type <
 
 After the connection is established and the device is bonded, you can use the mouse with the connected device.
 
+Windows HLK tests
```
Comment (greg-fer) [inline applications/nrf_desktop/README.rst:791]:
```suggestion
Windows Hardware Lab Kit tests
```

File: applications/nrf_desktop/README.rst [hunk 3]
```diff
@@ -788,6 +788,12 @@ After building the application with or without :ref:`specifying the build type <
...
+-----------------
+
+The nRF Desktop devices are confirmed to pass the tests from official playlist (:file:`HLK Version 1903 CompatPlaylist x86 x64 ARM64.xml`) required for compatibility with Windows 10 by Windows Hardware Compatibility Program.
```
Comment (greg-fer) [inline applications/nrf_desktop/README.rst:794]:
```suggestion
The nRF Desktop devices have passes the tests from official playlist required for compatibility with Windows 10 by Windows Hardware Compatibility Program (:file:`HLK Version 1903 CompatPlaylist x86 x64 ARM64.xml`).
```

**Review by greg-fer — CHANGES_REQUESTED** [review]
Some suggestions.

File: applications/nrf_desktop/README.rst (hunk 3 above)
Comment (MarekPieta) [reply applications/nrf_desktop/README.rst:794]: `have passed` ?

File: applications/nrf_desktop/README.rst (hunk 1 above)
Comment (MarekPieta) [reply applications/nrf_desktop/README.rst:797]: Ok

**Review by greg-fer — APPROVED** [review]

**Review by shantha-14 — APPROVED** [review]

**Pull request merged**